src/game/level.h

```diff
@@ -14,17 +15,53 @@
 #include <string>
 #include <vector>
 
+struct Background {
+  i64 x;
+  i64 y;
+  std::vector<u32> background_data;
+};
+
+struct Objects {
+  i64 x;
+  i64 y;
+  u32 width;
+  u32 height;
+  u32 tileset_id;
+  std::string script;
+};
+
+struct Collisions {
+  i64 x;
+  i64 y;
+  std::vector<u64> collisions_data;
```

Comment:
`std::vector` should be at the very top of the struct definition, as its the largest member variable (24 bytes / 192 bits)